generators/app/templates/test/__mocks__/solid-auth-client.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
Is there a way to remove the need for this disable?

Reply:
Removed